### PR TITLE
correct inaccurate bird information

### PR DESCRIPTION
**Pull request by bprcode**
Penguins are much cooler than flamingos and should be acknowledged as such.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # silver-octo-chainsaw
 Practice managed repository
```
Comment (bprcore) [inline README.md:2]: Might need a better title

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-This is a sample readme file in a repository which will be used to practice pull requests.
+This is a simple readme file in a repository which is being used to practice pull requests.
 
-Flamingos are the best feathered animals.
```
Comment (bprcore) [inline README.md:6]: Cannot fully neglect flamingos

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # silver-octo-chainsaw
 Practice managed repository
 
-This is a sample readme file in a repository which will be used to practice pull requests.
+This is a simple readme file in a repository which is being used to practice pull requests.
```
Comment (bprcore) [inline README.md:5]: Suggest more obnoxious commenting style

**Review by bprcore — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bprcode) [reply README.md:2]: Like what?

Comment (bprcore) [reply README.md:2]: "A Tail of Two Birds"

File: README.md (hunk 3 above)
Comment (bprcode) [reply README.md:5]: Disagree about this.

File: README.md (hunk 1 above)
Comment (bprcode) [reply README.md:2]: That's a great title, Me. I've made the change.

File: README.md (hunk 2 above)
Comment (bprcode) [reply README.md:6]: Agreed. Flamingos have their place as well.

**Review by bprcore — APPROVED** [review]
Allowing these changes to proceed now that flamingos and penguins are both represented.

**Pull request merged**